parsimonious/grammar.py

```diff
@@ -296,64 +298,74 @@ def __init__(self, custom_rules=None):
         """
         self.custom_rules = custom_rules or {}
 
-    def visit_parenthesized(self, parenthesized, (left_paren, _1,
-                                                  expression,
-                                                  right_paren, _2)):
+    def visit_parenthesized(self, parenthesized, _a):
```

Comment:
I updated the naming as above in dd580437b952a607dde39d0a9c7651f0f0cd1233. I'm happy to revert if you'd prefer a different naming scheme.